src/app/components/players/Cards.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Member from './Member';
+
+//export const Cards: React.FC = React.memo(() => {
+class Cards extends React.Component {
+    state = {
+        users: [
+            {
+                name: 'Jhon Doe',
+                status: true,
+            },
+            {
+                name: 'Ragnar Lothbrok',
+                status: false,
+            },
+        ],
+    };
+
+    createUser = () => {
+        const member = [];
+
+        for (let idx = 0; idx < this.state.users.length; idx++) {
+            const nickname = this.state.users[idx].name;
+            const online = this.state.users[idx].status;
+
+            member.push(<Member name={nickname} status={online} />);
```

Comment:
Dernière chose qui need absolument d'être corrigé c'est que vos ``Members`` n'ont pas de ``key`` et React va nous balancer des erreurs aléatoirement. Lorsque vous rendez une liste, toujours assigné des ``key`` aux childs elements. Donc ce que je vous conseil de faire c'est de rajouter un champ ``id`` à chaque user et d'utiliser le map que je vous ai filé au dessus 

https://fr.reactjs.org/docs/lists-and-keys.html